OpenSim/Simulation/Manager/Manager.h

```diff
@@ -95,38 +95,29 @@ class OSIMSIMULATION_API Manager
     double _ti;
     /** Final time of the simulation. */
     double _tf;
-    /** First dt in an integration. */
-    double _firstDT;
     
     /** Storage for the states. */
     std::unique_ptr<Storage> _stateStore;
 
-   int _steps;
-   /** Number of integration step tries. */
-   int _trys;
-   /** Maximum number of steps in an integration. */
-   int _maxSteps;
-   /** Flag for signaling a desired halt. */
-   bool _halt;
-   /** Minimum step size. */
-   double _dtMin;
```

Comment:
Will do. It does seem that `_dtMax` affects the local variable `dt`, but then `dt` actually has no effect on the integration itself. I think that `dt` could be removed safely, and further, the input variable `dtFirst` could also be removed and the signature can be changed safely (searching through the code, I only see uses of `Manager::integrate(state)`. @aseth1 would you be in favor of removing `dt` and changing the signature to remove `dtFirst` as well?